packages/superset-ui-plugin-chart-icicle/types/external.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.png' {
```

Comment:
What's this going to be used for? I've never actually seen a declaration of a filetype module.
(Not a change request -- just wondering what the pattern was)

Reply:
It's a pattern that I saw the other superset-ui-plugins follow --- it's used to create a chart's metadata which includes a thumbnail image of what an example plugin of this type would look like